variables-skyline/index090.html

```diff
@@ -59,5 +59,5 @@
 </html>
 
 <!--
-  Add these properties to your `bb1` class: `display: flex;`, `flex-direction: column;`, and `align-items: center;`. This will center the parts of the building using "flex" or "flexbox". You will learn about it in more detail on another project.
+  Give your `bb1` element these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`. This will center the parts of the building using "flex" or "flexbox". You will learn about it in more detail on another project.
```

Comment:
The description is fine, but this could be split up with brief descriptions about what each property will do. This might be the first time students use Flexbox. Speaking of which, I forgot to capitalize Flexbox here before force pushing.